### DISCUSSION TITLE
useFormContext typescript

### DISCUSSION THREAD

**Discussion by olesmarola** [react-hook-form/react-hook-form#4828] [Q&A]
Hello! I got typescript error when use useFormContext hook:
here is my useForm: https://cln.sh/z9aiAm
formProvider: https://cln.sh/NkK5sM
type for fields: https://cln.sh/EqsqdI
and when I use formContext, I got this: https://cln.sh/1XTSQ8

Maybe I do something wrong? Can you help me, please?

**Comment by bluebill1049** [comment]
What's your form `type`?
